components/raftstore/src/store/fsm/store.rs

```diff
@@ -2728,6 +2734,7 @@ impl<'a, EK: KvEngine, ER: RaftEngine, T: Transport> StoreFsmDelegate<'a, EK, ER
                 stats.set_damaged_regions_id(damaged_regions_id);
             }
             completed_apply_peers_count = meta.completed_apply_peers_count;
```

Comment:
Is it equal to region_count - pending_apply_peers_count?

Reply:
Nope, this counter records the accurate count of regions which complete on applying pending raft logs.

As mentioned above in https://github.com/tikv/tikv/pull/16239#discussion_r1470831161, if the tick is not fresh as expected, the `region_count - pending_apply_peers_count` will contains the regions which has not been collected infos on whether it's pending on applying raft logs.

Reply:
Better add a comment

Reply:
Add extra annotations to the specification of `completed_apply_peers_count`.